repository-ydb-v2/src/test/java/tech/ydb/yoj/repository/ydb/merge/QueriesMergerTest.java

```diff
@@ -154,7 +156,9 @@ private <T extends Entity<T>> YdbRepository.Query<?> insert(T p) {
 
     @SuppressWarnings("unchecked")
     private <T extends Entity<T>> YdbRepository.Query<?> find(T p) {
-        return new YdbRepository.Query<>(YqlStatement.find((Class<T>) p.getClass()), p.getId());
+        EntitySchema<T> schema = EntitySchema.of((Class<T>) p.getClass());
```

Comment:
minor: style: `var schema` is preferable here, I think

Reply:
I prefer var only in new statements. It's little bit more symbols, but more understandable in RO mode, i think :)